tasks/opsep/update-l2pao-to-aliased-l1pao/VALIDATION.md

```diff
@@ -0,0 +1,77 @@
+# Validation
+
+This document can be used to validate the state diff resulting from the execution of the upgrade
+transaction.
+
+For each contract listed in the state diff, please verify that no contracts or state changes shown in the Tenderly diff are missing from this document. Additionally, please verify that for each contract:
+
+- The following state changes (and none others) are made to that contract. This validates that no unexpected state changes occur.
+- All addresses (in section headers and storage values) match the provided name, using the Etherscan and Superchain Registry links provided. This validates the bytecode deployed at the addresses contains the correct logic.
+- All key values match the semantic meaning provided, which can be validated using the storage layout links provided.
+
+## State Overrides
+
+The following state overrides should be seen:
+
+### `0xb41890910b05dCba3d3dEF19B27E886C4Ab406EB` (The 1 of 1 `ProxyAdmin` owner Safe on L2)
```

Comment:
What is this `0xb41890910b05dCba3d3dEF19B27E886C4Ab406EB` address? I know it's the current L2PAO on OP Sepolia, and it seems to be a 1/1, but who is the owner?

Is this 1/1 Safe used elsewhere in the protocol on OP Sepolia? If so we should PR this into https://github.com/ethereum-optimism/safe-wallets

Also I thought you said the L2PAO is an EOA, was that changed to a 1/1 safe at some point? The 1/1 safe owner is 0xfd1D2e729aE8eEe2E146c033bf4400fE75284301, the same one you mentioned was the EOA owner

It would be great if we can provide all the context that answers these questions in the README

Reply:
Added this information to the readme: https://github.com/ethereum-optimism/superchain-ops/pull/204/commits/72e34f6e68461051d9422969540ff7806f96a20a